openapi/components/schemas/schedules/GetListOfSchedulesResponse.yaml

```diff
@@ -1,9 +1,9 @@
-title: Getlistofschedulesresponse
+title: GetListOfSchedulesResponse
 required:
   - data
 type: object
 properties:
   data:
     allOf:
-      - $ref: ./data24.yaml
+      - $ref: ./ScheduleFilters.yaml
```

Comment:
the `allOf` could be removed, or are you planning to do that all at once?

Reply:
Yeah probably. Didn't touch this on previous PRs.